homeassistant/components/influxdb.py

```diff
@@ -65,7 +66,7 @@
         vol.Optional(CONF_PORT): cv.port,
         vol.Optional(CONF_SSL): cv.boolean,
         vol.Optional(CONF_RETRY_COUNT, default=0): cv.positive_int,
-        vol.Optional(CONF_RETRY_QUEUE, default=20): cv.positive_int,
+        vol.Optional(CONF_RETRY_QUEUE, default=100): cv.positive_int,
```

Comment:
I don't think the name now indicates what this is. I would now name this just `CONF_EVENT_QUEUE_SIZE`. But that would be a breaking change, so it's not necessary I think.

Reply:
I think @MartinHjelmare is right and we should make a breaking change...

Reply:
Yeah, I agree that the naming is off but I did not want to make the change because I actually don't think users should worry about configuring a queue size.

I will rework it to drop entries based on time instead.